java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/WorkfolwExample.java

```diff
@@ -0,0 +1,60 @@
+package com.nedap.retail.services.examples;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.joda.time.DateTime;
+
+import com.nedap.retail.messages.Client;
+import com.nedap.retail.messages.workflow.WorkflowEvent;
+import com.sun.jersey.api.client.UniformInterfaceException;
+
+public class WorkfolwExample {
+
+    private static final String LOCATION = "http://location-testing";
+
+    public static void runExample(final Client client) {
+        System.out.println("-------------");
+        System.out.println("Workflow API example");
+        System.out.println("-------------");
+
+        final WorkflowEvent workflow = makeWorkflowEvent();
+
+        try {
+            // capture
+            System.out.println("------------- Capture workflow events");
```

Comment:
It's for the aesthetics, to differentiate "status" messages from output data. But until I menage to actually run the example I can not  tell you is it very useful.